gym_solo/core/rewards.py

```diff
@@ -374,7 +374,7 @@ def compute(self) -> float:
 
 
 def tolerance(x: float, bounds: Tuple[float, float] = (0., 0.), 
-              margin: float = 0., margin_value: float = 1e-6):
+              margin: float = 0., margin_value: float = .1):
```

Comment:
Experimentally found-- `1e-6` was too small. Keeping it a magic number rn cause I'm writing code to phase this function out regardless.